doc/asciidoc/connected-components.adoc

```diff
@@ -243,32 +247,30 @@ include::scripts/connected-components.cypher[tag=cypher-loading]
 
 `algo.unionFind`
 
-- if a threshold configuration parameter is supplied only relationships with a property value higher then the threshold
-are merged
+* If a threshold configuration parameter is supplied, only relationships with a property value higher than the threshold
+are merged.
 
 `algo.unionFind.queue`
 
-- parallel UnionFind using ExecutorService only.
-- Algorithm based on the idea that DisjointSetStruct can be built using just a partition of the nodes
-which are then merged pairwise.
-- The implementation is based on a queue which acts as a buffer for each computed DSS. As long as there are
-more elements on the queue the algorithm takes two, merges them and adds its result to the queue until only
-1 element remains.
+* Parallel `UnionFind`, using `ExecutorService` only.
+* Algorithm based on the idea that `DisjointSetStruct` can be built using just a partition of the nodes, which are then merged pairwise.
+* The implementation is based on a queue which acts as a buffer for each computed `DisjointSetStruct`.
+As long as there are more elements on the queue, the algorithm takes two, merges them, and adds its result to the queue until only 1 element remains.
 
 `algo.unionFind.forkJoinMerge`
 
--  Like in *exp1* the resulting DSS of each node-partition is merged by the ForkJoin pool while
-the calculation of the DSS is done by the ExecutorService.
+* Like in *exp1* the resulting `DisjointSetStruct` of each node-partition is merged by the `ForkJoin` pool, while
```

Comment:
@mneedham I'm not sure what "*exp1*" was in reference to?

Reply:
the *exp1* is experimental version one of the algorithm that was renamed to algo.unionFind.queue probably just judging by the sequence in docs